### PR TITLE
Update introduction sample notebook for openCypher

### PR DESCRIPTION
**Pull request by michaelnchin**
Issue #, if available: N/A

Description of changes:
- Updated `01-About-the-Neptune-Notebook` with openCypher magic query examples and sample application notebooks.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb [hunk 1]
```diff
@@ -256,6 +259,107 @@
...
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Similar to the other query language magics, `%%oc` supports retrieval of a query explain plan."
```
Comment (joywa) [inline src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb:307]: Do we need to change this? The query below is not for explain, but for bolt

File: src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb [hunk 2]
```diff
@@ -269,14 +373,21 @@
...
     "[Explore English Premier League using SPARQL](../02-Visualization/EPL-SPARQL.ipynb)\n",
     "\n",
+    "[Explore English Premier League using openCypher](../02-Visualization/EPL-openCypher.ipynb)\n",
+    "\n",
     "Curious about the business problems can be solved with graph? Check out these sample application notebooks for some inspiration.\n",
     "\n",
     "[Introduction to Fraud Graphs](../03-Sample-Applications/01-Fraud-Graphs/01-Building-a-Fraud-Graph-Application.ipynb)\n",
     "\n",
-    "[Introduction to Knowledge Graphs](../03-Sample-Applications/02-Knowledge-Graphs/01-Building-a-Knowledge-Graph-Application.ipynb)\n",
+    "[Introduction to Knowledge Graphs](../03-Sample-Applications/02-Knowledge-Graphs/01-Building-a-Knowledge-Graph-Application-Gremlin.ipynb)\n",
```
Comment (joywa) [inline src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb:382]: Suggesting this link instead, which lets user choose whether they want the Gremlin or OC version: https://github.com/aws/graph-notebook/tree/main/src/graph_notebook/notebooks/03-Sample-Applications/02-Knowledge-Graphs

File: src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb [hunk 3]
```diff
@@ -269,14 +373,21 @@
...
     "\n",
-    "[Introduction to Identity Graphs](../03-Sample-Applications/03-Identity-Graphs/01-Building-an-Identity-Graph-Application.ipynb)"
-   ]
+    "[Introduction to Identity Graphs](../03-Sample-Applications/03-Identity-Graphs/01-Building-an-Identity-Graph-Application.ipynb)\n",
+    "\n",
+    "[Introduction to Security Graphs](../03-Sample-Applications/04-Security-Graphs/01-Building-a-Security-Graph-Application-with-Gremlin.ipynb)"
```
Comment (joywa) [inline src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb:386]: Same with this link, update to the higher level so users can choose based on language: https://github.com/aws/graph-notebook/tree/main/src/graph_notebook/notebooks/03-Sample-Applications/04-Security-Graphs

**Review by joywa — CHANGES_REQUESTED** [review]
Thanks for getting this turned around so quickly! Just a couple of small updates.

File: src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb (hunk 1 above)
Comment (michaelnchin) [reply src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb:307]: Removed this section.

File: src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb (hunk 2 above)
Comment (michaelnchin) [reply src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb]: Done, using the base folder link.

File: src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb (hunk 3 above)
Comment (michaelnchin) [reply src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb]: Done

**Pull request merged**